Package.swift

```diff
@@ -25,13 +25,13 @@ let package = Package(
         .library(name: "SpeziStudyDefinition", targets: ["SpeziStudyDefinition"])
     ],
     dependencies: [
-        .package(url: "https://github.com/apple/FHIRModels.git", .upToNextMinor(from: "0.5.0")),
-        .package(url: "https://github.com/StanfordSpezi/Spezi.git", .upToNextMajor(from: "1.8.1")),
-        .package(url: "https://github.com/StanfordSpezi/SpeziFoundation.git", .upToNextMajor(from: "2.1.4")),
-        .package(url: "https://github.com/StanfordSpezi/SpeziHealthKit.git", .upToNextMajor(from: "1.1.0")),
+        .package(url: "https://github.com/apple/FHIRModels.git", from: "0.5.0"),
```

Comment:
Probably even makes sense to bump it to 0.7.0 to ensure that we force the concurrency fixed across all packages?